packages/remix-react/components.tsx

```diff
@@ -295,6 +296,10 @@ function usePrefetchBehavior(
  */
 let NavLink = React.forwardRef<HTMLAnchorElement, RemixNavLinkProps>(
   ({ to, prefetch = "none", ...props }, forwardedRef) => {
+    let location = typeof to === "string" ? to : createPath(to);
+    let isAbsolute =
+      /^[a-z+]+:\/\//i.test(location) || location.startsWith("//");
```

Comment:
I don't think this occurred to me in RR, but we should only need to check the regex if they passed in a string - since you can't specify an absolute URL using the object syntax (`<Link to={{ pathname: 'https://...' }}>`) - so could we avoid the need for `createPath`/`location` (which is already handled in `useHref`) and just do this?

```suggestion
    let isAbsolute = 
      typeof to === "string" &&
      (/^[a-z+]+:\/\//i.test(to) || to.startsWith("//"));
```

Reply:
ha, i was also thinking that myself a few minutes ago 😆